### DISCUSSION TITLE
Trying to move the menu into a separate file, so the tech-writers do not modify general config

### DISCUSSION THREAD

**Discussion by atastycookie** [squidfunk/mkdocs-material#3047] [Q&A] [answered]
Hello,

I have tried to move the menu into a separate file so the tech-writers do not modify the general config. For that, I have used INHERIT construction [https://www.mkdocs.org/user-guide/configuration/#configuration-inheritance](https://www.mkdocs.org/user-guide/configuration/#configuration-inheritance), but I faced the error:
```
WARNING -  Config value: 'INHERIT'. Warning: Unrecognised configuration name: INHERIT
```

How I do that

1. Inside the `mkdocs.yml` file, I put the `INHERIT: menu.yml` at the start of the file and remove `nav:` block
2. Create the new one file `menu.yml` in the same directory as the `mkdocs.yml`
3. Run build `sudo docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material`
4. Face that menu is "removed", and my navigation is not used :/

What did I do wrong?

**Comment by squidfunk** [comment, marked as answer]
Which version of MkDocs are you running? [Configuration inheritance](https://www.mkdocs.org/about/release-notes/#support-added-for-configuration-inheritance-2218) is available as of MkDocs 1.2.

**Reply by atastycookie** [reply]
As I think, the latest from the `squidfunk/mkdocs-material` (using the Docker from official documentation).

UPD. Oops, I did update it via `docker pull squidfunk/mkdocs-material`, and now I have the latest version, and the newer image supports it. Sorry for that.

**Reply by squidfunk** [reply]
No worries, I adjusted my question to include the answer 😉
